conf/variant/common/local.conf

```diff
@@ -4,6 +4,9 @@ DL_DIR = "${TOPDIR}/downloads"
 SDKMACHINE = "x86_64"
 DISTRO = "pelux"
 
+# Target Static IP address
```

Comment:
A better comment would be nice here. Why is this a good thing, what is it used for etc.

Reply:
I added the comment now.